db/m_seed.rb

```diff
@@ -1,7 +1,7 @@
 mimizon = Store.create(name: "Mimizon", description: "You want all the books? You can have all the books", image: "mimizon.png")
 indiecovers = Store.create(name: "Indie Covers", description: "A collection of independent titles aimed for hipsters", image: "indie-covers.png")
 
-books = Category.create(name: "books", title: "Read to your heart's content")
```

Comment:
Looks like you are setting the same attribute twice here.

Reply:
I removed that bit. Categories is just down to title now